boards/FSM/Src/main.c

```diff
@@ -206,63 +204,43 @@ int main(void)
 
     Io_SharedHardFaultHandler_Init();
 
-    Io_FlowMeters_Init(&htim4);
-    flow_meter_in_range_check =
-        App_InRangeCheck_Create(Io_FlowMeters_GetFlowRate, MIN_FLOW_RATE_L_PER_MIN, MAX_FLOW_RATE_L_PER_MIN);
-
-    Io_WheelSpeedSensors_Init(&htim16, &htim17);
-    left_wheel_speed_sensor_in_range_check = App_InRangeCheck_Create(
```

Comment:
Can we fold the in-range checks into the App files as part of the struct? This would clean up main a good deal, and more closely couple related logic to a given sensor

Reply:
I believe that is what I did?